.circleci/config.yml

```diff
@@ -1,29 +1,25 @@
-version: 2
-
+version: 2.1
 defaults: &defaults
   docker:
     - image: circleci/golang:1.16
-
 workflows:
   version: 2
   build-and-test:
     jobs:
       - test
       - build
-
 jobs:
-
   test:
-    << : *defaults
+    !!merge <<: *defaults
     steps:
       - checkout
       - run: make test
-
   build:
-    << : *defaults
+    !!merge <<: *defaults
     steps:
       - checkout
-      - run: make bin/ssm-env
+      # Compile, and make sure it's not dynamically linked.
+      - run: CGO_ENABLED=0 make bin/ssm-env && ! ldd bin/ssm-env
```

Comment:
What do you think about just putting this in the Makefile instead?

Reply:
Lol, just had that thought. Okay, I'll go ahead and do that.